tracer.go

```diff
@@ -24,35 +24,45 @@ import (
 	"net/http"
 )
 
-/**
-Will be injecting the trace id and span id  in the logger.
-*/
-func InjectTraceInformationInLogger() logginghttp.LoggerFunc {
+// Will be injecting the trace id and span id  in the logger.
```

Comment:
This comment doesn't follow Effective Go guidelines. Here is an example of how I would fix it:
```suggestion
// InjectTraceInformationInLogger adds the trace id and span id to key value pairs that can 
// be provided to a logger.
```

Reply:
Modified comments as suggested.